### PR TITLE
Autobump mecanism

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
📦 :
- Implement a command to take care of the auto-bumping of a package
- Set up a sub command system.
- And also `scaffold`/`setup` command, and `validate`.

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
available as of `@coorpacademy/update-node@3.0.0-alpha.4` 😉

File: src/bump-dependencies.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        )
+        .join('\n')}\n`
+    }
+  };
+};
+
+const commitAndMakePullRequest = config => async options => {
+  const {branch, message, pullRequest} = options;
+
+  if (!config.baseBranch) return Promise.resolve();
```
Comment (Fausto95) [inline src/bump-dependencies.js]: should reject nah?

File: src/bump-version.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  if (tooShortMessage) return PATCH;
+  return MINOR;
+};
+
+const getReleaseType = async selectionCommand => {
+  const releaseType = selectionCommand
+    ? await new Promise((resolve, reject) => {
+        const res = shelljs.exec(selectionCommand);
+        if (res.code === 0) return res.stdout;
+        throw makeError('Fail to get release type', {
```
Comment (Fausto95) [inline src/bump-version.js]: Failed

File: src/bump-dependencies.js (hunk 1 above)
Comment (AdrieanKhisbe) [reply src/bump-dependencies.js]: bien vu

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**